module/entities/helper.js

```
@@ -231,4 +231,100 @@ module.exports = class EntitiesHelper {
+						message: CONSTANTS.apiResponses.PROJECT_NOT_FOUND,
+					}
+				}
+				// Fetch entity details from entity-service
+				const entityDetails = await entityManagementService.entityDocuments({
+					_id: entityId,
+					tenantId: tenantId,
+				})
+				// Throw error if the entity details are not fetched
+				if (!entityDetails?.success || !entityDetails?.data.length > 0) {
+					throw {
+						status: HTTP_STATUS_CODE.bad_request.status,
```

ESLint

[error] 'HTTP_STATUS_CODE' is not defined.

(no-undef)